cli/root.go

```diff
@@ -58,7 +59,42 @@ func init() {
 	cobra.AddTemplateFuncs(templateFunctions)
 }
 
-func Root() *cobra.Command {
+func CoreSubcommands() []*cobra.Command {
```

Comment:
Given the package name of CLI, commands would be implicit. I might call this `Core` instead.